src/main/java/uk/gov/pay/ledger/queue/EventMessageDto.java

```diff
@@ -21,19 +22,24 @@ public class EventMessageDto {
 
     private Boolean live;
 
+    @NotNull
```

Comment:
Can we add test(s) for these changes?

Reply:
have added an integration test to demonstrate that a request missing one of the mandatory fields will be rejected with 422. Is this sufficient?